portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,13 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Cycles through 3 different pictures of myself.
+let pictureNumber = 0;
+function cyclePictures() {
+  let headerSelfie = document.getElementById("header-selfie");
+  const imageNames = ["campus", "suit", "mainSelfie"];
+  headerSelfie.src = "images/" + imageNames[pictureNumber] + ".jpg";
```

Comment:
Just an FYI. Dynamically loading resources by name (mostly scripts, files, etc. but sometimes images) might not work once resources start to be bundled and minified.

No action neeeded.

Reply:
Noted.